docs/install_guides/pypi-openvino-dev.md

```diff
@@ -54,17 +54,15 @@ To avoid dependency conflicts, use a virtual environment. Skip this
 
 Create virtual environment:
 
-On Linux and macOS:
-```sh
-# Depending on your OS, this step may require installing python3-venv
-python3 -m venv openvino_env
-```
-
-On Windows: 
 ```sh
+python -m pip install --user virtualenv 
```

Comment:
Do you know if this is sufficient on a clean install of Ubuntu 18 and 20 (after installing pip, as in the note below), or do we have user acceptance testers that will test this on these systems?

Reply:
Added instruction to install pip for Ubuntu

Reply:
Added instruction to install pip for Ubuntu.